service_user.go

```diff
@@ -32,6 +32,14 @@ type (
 		RedisACLCommands         []string `json:"redis_acl_commands"`
 		RedisACLKeys             []string `json:"redis_acl_keys"`
 		RedisACLChannels         []string `json:"redis_acl_channels"`
+		ValkeyACLCategories      []string `json:"valkey_acl_categories"`
+		ValkeyACLCommands        []string `json:"valkey_acl_commands"`
+		ValkeyACLKeys            []string `json:"valkey_acl_keys"`
+		ValkeyACLChannels        []string `json:"valkey_acl_channels"`
+		DragonflyACLCategories   []string `json:"dragonfly_acl_categories"`
+		DragonflyACLCommands     []string `json:"dragonfly_acl_commands"`
+		DragonflyACLKeys         []string `json:"dragonfly_acl_keys"`
+		DragonflyACLChannels     []string `json:"dragonfly_acl_channels"`
```

Comment:
Hey. I think there is no `dragonfly_acl_channels` https://github.com/aiven/go-client-codegen/blob/64158e7f773879a78ce561ee36d56c9095b52228/handler/service/service.go#L820

Reply:
Ah great catch, updated.